packages/notification-center/src/components/notification-center/components/FeedsTabs.tsx

```diff
@@ -62,34 +57,10 @@ const TabLabelWrapper = styled.div`
 `;
 
 function UnseenBadgeContainer({ storeId }: { storeId: string }) {
-  const { api } = useApi();
-  const { stores } = useFeed();
-  const { unseenCount: generalUnseenCount } = useUnseenCount();
-
-  const [unseenCount, setUnseenCount] = useState<number>();
-
-  useEffect(() => {
-    setCount(stores, storeId, api, setUnseenCount);
-  }, [generalUnseenCount]);
+  const { stores } = useNotifications();
+  const query = useMemo(() => stores?.find((i) => i.storeId === storeId)?.query || {}, [stores]);
```

Comment:
In order to fix your count issue you need to return the seen=false. 

```suggestion
  const query = useMemo(() => stores?.find((i) => i.storeId === storeId)?.query || {}, [stores]);
  query = Object.assign({}, query, { seen: false });

```

Maybe it will be a good idea to update this one to be false as defualt
https://github.com/novuhq/novu/blob/8a45791436721808b1447d032aa02de3f85c53aa/apps/api/src/app/widgets/widgets.controller.ts#L127
but if you ask me i think we should remove it at all from this end point because this one is count and not unseen.

Reply:
@djabarovgeorge to me this thing `query = Object.assign({}, query, { seen: false })` feels hacky, because we will override what the user has provided. I thought that if I won't provide `seen` at all it will return to me all the notifications from the particular feed.

Reply:
Well in this context it makes sense to add the seen false because it is unseen count component in the future I believe we will get the option to toggle from unseen/unread, however, as I said in the comment above I think we should remove the default seen=true from `/count` route in the API (and then you should get the behavior you were talking about)